README.md

```diff
@@ -76,50 +76,3 @@ The `ctf/final-ctf/challenge-folder-1/` and `ctf/mid-ctf/challenge-folder-1/` fo
 `-- src/
     `-- index.template.php
 ```
-
```

Comment:
Why is this gone?

Reply:
In this file it's still a new line but at the end of all removed lines.